### PR TITLE
Only add the parentId if it has been explicitly specified

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: shared/gh/api/gh.api.orgunit.js [hunk 1]
```diff
@@ -114,17 +114,24 @@ define(['exports'], function(exports) {
         // Set a default callback function in case no callback function has been provided
         callback = callback || function() {};
 
+        // Request options object
+        var data = {
+            'app': appId,
+            'displayName': displayName,
+            'type': type,
+            'groupId': groupId,
```
Comment (bp323) [inline shared/gh/api/gh.api.orgunit.js]: Should we make `groupId` and `description` optional as well?

**Comment by bp323** [comment]
Small question posted. To @Coenego for followup

**Comment by Coenego** [comment]
Reassigning to @bp323 for review after follow-up

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**